x-pack/plugins/reporting/server/export_types/csv_searchsource/csv_searchsource.ts

```diff
@@ -54,22 +52,12 @@ export class CsvSearchsourceExportType extends ExportType<
     LICENSE_TYPE_PLATINUM,
     LICENSE_TYPE_ENTERPRISE,
   ];
-  declare startDeps: CsvSearchsourceExportTypeStartDeps;
-
   constructor(...args: ConstructorParameters<typeof ExportType>) {
     super(...args);
     const logger = args[2];
     this.logger = logger.get('csv-searchsource-export');
   }
 
-  setup(setupDeps: ExportTypeSetupDeps) {
-    this.setupDeps = setupDeps;
-  }
-
-  start(startDeps: CsvSearchsourceExportTypeStartDeps) {
-    this.startDeps = startDeps;
-  }
-
```

Comment:
Take note that the export type sub-classes should not need to implement these functions. When callers use `exportType.setup()` and `exportType.start()`, the call goes straight to the implementation in the super class.